api/db/authorization.js

```diff
@@ -34,19 +34,19 @@ module.exports = () => ({
       return this.related('activities').pluck('name');
     },
 
-    async validate(activities, model = this, activityModel) {
+    async validate({ transacting, activities = [] } = {}) {
```

Comment:
The base model passes a single argument to `validate` with a possible `transacting` property.  If the `validate` method needs additional arguments, we can make them properties of the "one" object.

In most cases, extra arguments are unnecessary because you validate the model's "new" attributes, but this `roles` model is quirky because there's no way to get the new activities from the role model itself until the role is saved, but we don't want to save it until it's validated.  🤷‍♂️